<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>USB Relay Controller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Reset_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="COM_Name">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="COM_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>23</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ON_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OFF_Button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Delay_Value">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Delay_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>63</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Delay</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_s">
    <property name="geometry">
     <rect>
      <x>182</x>
      <y>63</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>143</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHELP">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuHELP"/>
  </widget>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
